mobile/contexts/auth/AuthContext.provider.tsx

```diff
@@ -28,8 +30,10 @@ const AuthContextProvider = ({ children }: React.PropsWithChildren) => {
       });
       SecureStore.setItem("access_token", token);
       setIsAuthenticated(true);
+      setAuthError(false);
     } catch (err: unknown) {
       console.log("Error while trying to sign in", err);
+      setAuthError(true);
```

Comment:
instead of this, we can throw the error that we catch here, so we will be able to access it in the login

Reply:
Done